### PR TITLE
Fixes for query. Added simple dump.

### PR DESCRIPTION
**Pull request by jaynabonne**

### CODE REVIEW

File: Assets/Scripts/DemoQuadTree.gd [hunk 1]
```diff
@@ -51,7 +52,7 @@ func _ready() -> void:
 	root_qt_node.draw(0.2)
 	
 
-func _physics_process(delta: float) -> void:
+func _process(delta: float) -> void:
```
Comment (sairam4123) [inline Assets/Scripts/DemoQuadTree.gd:55]: Well, why did you change the _physics_process, tho?

File: addons/sairam.quadtree/QuadTree.gd [hunk 2]
```diff
@@ -155,23 +155,21 @@ func _query(bound: AABB) -> Array:
 	_found_objects.clear()
 	for object in _objects:
 		var transformed_aabb = object.get_transformed_aabb()
-		if transformed_aabb != bound and bound.intersects(transformed_aabb):  # check if the object in the bounds and it's not bound
```
Comment (sairam4123) [inline addons/sairam.quadtree/QuadTree.gd]: This part is available in C++ translation, but I think it's not necessary.

File: Assets/Scripts/DemoQuadTree.gd (hunk 1 above)
Comment (jaynabonne) [reply Assets/Scripts/DemoQuadTree.gd:55]: I can change it back if you like. The code didn't seem to have much to do with physics, so I had changed it when trying to work out what was going on.

Comment (sairam4123) [reply Assets/Scripts/DemoQuadTree.gd:55]: Ready to be merged.
